spec/models/organization_spec.rb

```diff
@@ -240,14 +240,22 @@
 
       it "updates the paths of the organization's articles" do
         new_slug = "slug_#{rand(10_000)}"
-        organization.update(slug: new_slug)
+
+        sidekiq_perform_enqueued_jobs(only: Organizations::UpdateOrganizationArticlesPathsWorker) do
```

Comment:
no idea why this is currently working in master as it shouldn't since it didn't execute the jobs needed for the to pass 🤷